governance/handler.go

```diff
@@ -445,8 +441,7 @@ func (gov *Governance) GetGovernanceItemAtNumber(num uint64, key string) (interf
 	}
 }
 
-func (gov *Governance) GetLatestGovernanceItem(key string) interface{} {
-	gov.currentSetMu.RLock()
-	defer gov.currentSetMu.RUnlock()
-	return gov.currentSet[key]
+func (gov *Governance) GetLatestGovernanceItem(key int) interface{} {
+	ret, _ := gov.currentSet.GetValue(key)
```

Comment:
`GetValue` holds the own lock, but the function checks existence in GovernanceKeyMapReverse without a  lock. Is there no race in the method?

```
func (gs *GovernanceSet) GetValue(key int) (interface{}, bool) {
	sKey, ok := GovernanceKeyMapReverse[key]
	if !ok {
		return nil, false
	}
...
```

Reply:
@jeonghonah GovernanceKeyMapReverse is a map which can't be written online. So all access to the map is "READ" and no race can happen on it.